wlauto/workloads/youtube/__init__.py

```diff
@@ -0,0 +1,123 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import logging
+import re
+import time
+
+from wlauto import AndroidUiAutoBenchmark, Parameter
+
+
+class Youtube(AndroidUiAutoBenchmark):
+
+    name = 'youtube'
+    package = 'com.google.android.youtube'
+    activity = ''
+    description = '''
+    A workload to perform standard productivity tasks within YouTube.
+
+    The workload plays a pre-defined video from the Watch Later playlist in different video
+    quality bitrates.  It also seeks to a pre-defined position, pauses and restarts playback.
+
+    This workload requires a stable internet connection, preferably Wi-Fi.
```

Comment:
We can lose these types of comments.  They break the flow of what you are trying to say and I would hope it's implied anyway.